python/src/aiconfig/editor/client/src/LocalEditor.tsx

```diff
@@ -174,6 +196,14 @@ export default function Editor() {
     return await ufetch.get(ROUTE_TABLE.SERVER_STATUS);
   }, []);
 
+  const logEvent = useCallback((event: LogEvent, data?: LogEventData) => {
+    try {
+      datadogLogs.logger.info(event, data);
```

Comment:
So if telemetry is false, this is going to error? Instead of try catch I think just checking for non-null and no-op is better? We can stil wrap the actual `datadogLogs.logger.info(event, data)` call in a try-catch

Reply:
I manually tested this and this doesn't error, it just becomes a no-op